### DISCUSSION TITLE
Pending reviews being counted as submitted reviews according to search

### DISCUSSION THREAD

**Discussion by Nubly** [community/community#39698] [Enterprise]
### Select Topic Area

Bug

### Body

When filtering pull requests in the search bar based on `-reviewed-by:nubly`, the pull requests which I have started a review on but have not finished yet are actually counted as submitted reviews.

This has burned me a few times now, since I've bookmarked a page on my company's internal GitHub Enterprise instance which includes query strings that apply filters to the pull requests that I am looking at, one of these filters being the 'not reviewed by me' one mentioned earlier. This leads to me losing track of pull requests that I have already started a review on, as presumably a pending review is being counted as constituting a submitted review.

I wasn't sure whether or not to file this as an actual bug report, since pending results are also considered merge blockers in certain cases. It could be intentional behavior, I just would think that a pending review would not constitute a submitted review.

Additionally, I am unsure if this is unique to GitHub Enterprise or not. Please let me know if this is addressed already, as I was unable to find any previously documented information regarding it.

**Comment by ossanna16** [comment]
Hi there @Nubly and welcome to our community! Thank you for reporting this bug!
